src/routes/safe/components/Transactions/TxList/TxInfoDetails.tsx

```diff
@@ -89,7 +84,6 @@ export const TxInfoDetails = ({
         <AddressInfo address={address} name={name} avatarUrl={avatarUrl} />
         <EllipsisTransactionDetails
           address={address}
-          knownAddress={knownAddress}
```

Comment:
It wasn't needed?

Reply:
Instead of passing it as a prop I fetched the info where it was needed

https://github.com/gnosis/safe-react/pull/2352/files#diff-8522a88cc530096a38678da0be6ed5dab9b803a0a3d322b6b78577c4eafc9940R57